draft-ietf-quic-transport.md

```diff
@@ -2763,26 +2792,55 @@ Note:
   congestion control, which are not expected to be relevant for a closed
   connection. Retransmitting the final packet requires less state.
 
-New packets from unverified addresses could be used to create an amplification
-attack; see {{address-validation}}.  To avoid this, endpoints MUST either limit
-transmission of CONNECTION_CLOSE frames to validated addresses or drop packets
-without response if the response would be more than three times larger than the
-received packet.
+While in the closing period, an endpoint could receive packets from a new
+source address, indicating a connection migration; see {{migration}}. New
+packets from unverified addresses could be used to create an amplification
+attack; see {{address-validation}}. To avoid being used for an amplication
+attack, an endpoint in the closing state MUST either discard packets received
+from unvalidated addresses or limit the cumulative size of packets it sends to
+unvalidated addresses to 3 times the size of packets it receives from the
+address.
+
+An endpoint is not expected to handle key updates when it is closing. A key
+update might prevent the endpoint from moving from the closing state to
+draining, but it otherwise has no impact.
+
+
+### Draining Connection State {#draining}
+
+The draining state is entered once an endpoint receives a CONNECTION_CLOSE
+frame, which indicates that its peer is closing or draining. While otherwise
+identical to the closing state, an endpoint in the draining state MUST NOT send
+any packets. Retaining packet protection keys is unnecessary once a connection
+is in the draining state.
```

Comment:
We used to think that you could enter draining as a result of a stateless reset or idle timeout.  That is no longer the case, so this just mentions CONNECTION_CLOSE directly.